src/components/shared/Entity/Error/Logs.tsx

```diff
@@ -0,0 +1,45 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import {
+    Accordion,
+    AccordionDetails,
+    AccordionSummary,
+    Box,
+    Typography,
+} from '@mui/material';
+import Logs from 'components/Logs';
+import ErrorBoundryWrapper from 'components/shared/ErrorBoundryWrapper';
+
+export interface ErrorLogsProps {
+    logToken?: string | null;
+}
+
+function ErrorLogs({ logToken }: ErrorLogsProps) {
```

Comment:
Logs in the error panel can be collapsed by default to make a bit more room for the main view. Also, it allows users to control when they are fetching logs a bit better.